frontend/src/types.ts

```diff
@@ -935,6 +937,7 @@ export interface ChartParams {
 
 export interface DashboardItemLogicProps {
     dashboardItemId?: number | null
+    fromDashboardItemId?: number | null
```

Comment:
Nit: with saved insights coming into play, might a better name for this be `fromItemId`? I'm also confused about the difference between dashboardItemId and fromDashboardItemId. Maybe a comment here would help with that.

Reply:
Hey, made it much cleaner (I think) by renaming it from `DashboardItemLogicProps` to `SharedInsightLogicProps` and adding comments here.

![image](https://user-images.githubusercontent.com/53387/134646559-42385748-f458-40d2-8aff-cabd5dd77a8d.png)